scripts/jsonGenerator.sh

```diff
@@ -0,0 +1,111 @@
+# Commands, Inside the scripts folder
+# chmod +x ./jsonGenerator.sh
+# ./jsonGenerator.sh input.csv > output.json
```

Comment:
Please add the licensing headers as well.

Reply:
@GMishx added the license header.